notice_and_comment/static/regulations/css/less/comment-custom.less

```diff
@@ -8,26 +9,55 @@
 .activate-read {
   background: @blue;
   color: @white;
+
   &:hover {
     background-color: darken(@blue, 10%);
   }
 }
 
+.preamble-header {
+
+  .read-proposal,
+  .write-comment {
+
+    .read-icon,
+    .write-icon {
+      fill: @dark_text;
+    }
+
+    span {
+      padding-left: 5px;
+    }
+  }
+
+  .active-mode {
+
+    .read-icon,
+    .write-icon {
+      fill: @pacific;
```

Comment:
this fill should be `@blue` I believe. In general that's what I've been overriding any instances of `@pacific` from -site with